tests/integration/helpers/cluster.py

```diff
@@ -1221,6 +1255,22 @@ def setup_kafka_cmd(self, instance, env_variables, docker_compose_yml_dir):
         )
         return self.base_kafka_cmd
 
+    def setup_kafka_sasl_cmd(self, instance, env_variables, docker_compose_yml_dir):
+        self.with_kafka_sasl = True
+        env_variables["KAFKA_HOST"] = self.kafka_sasl_host
+        env_variables["KAFKA_EXTERNAL_PORT"] = str(self.kafka_sasl_port)
+        env_variables["KAFKA_DIR"] = instance.path + "/"
+        self.base_cmd.extend(
+            ["--file", p.join(docker_compose_yml_dir, "docker_compose_kafka_sasl.yml")]
+        )
+        self.base_kafka_sasl_cmd = self.compose_cmd(
+            "--env-file",
+            instance.env_file,
+            "--file",
+            p.join(docker_compose_yml_dir, "docker_compose_kafka_sasl.yml"),
+        )
+        return self.base_kafka_sasl_cmd
+
```

Comment:
This looks redundant

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/83581/commits/fd81d6c302afbfe8c9dd60c35ebca1f5bbc6b87b where I chose only the rabbitmq-related changes